frontend/src/modules/userManagement/views/userView.js

```diff
@@ -172,10 +172,18 @@ define(function(require){
     },
 
     onResetPasswordClicked: function(e) {
-      var $btn = $(e.currentTarget);
-      $btn.addClass('submitted');
-      Helpers.ajax('/api/createtoken', { email: this.model.get('email') }, 'POST', function() {
-        $btn.removeClass('submitted');
+      Origin.Notify.confirm({
+        text: Origin.l10n.t('app.confirmsendreset', { email: this.model.get('email') }),
```

Comment:
Completely missed that we could've avoided the `this` funk-ery altogether by just caching the email. Doesn't matter too much at this point!